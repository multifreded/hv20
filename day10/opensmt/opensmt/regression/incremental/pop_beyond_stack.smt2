(set-logic QF_LRA)
(check-sat)
(pop 1)

