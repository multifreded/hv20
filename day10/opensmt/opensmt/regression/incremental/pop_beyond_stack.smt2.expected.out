sat
(error "Attempt to pop beyond the top of the stack")

